nodejs assertion leads to proxy death

In our jupyter.cloudet.xyz deploy, we occassionally get the following crash that ultimately leads to proxy death. This is using the latest jupyter/configurable-http-proxy docker image from Docker Hub:

```
17:06:26.765 - info: [ConfigProxy] Proxying http://*:8000 to http://127.0.0.1:9999
17:06:26.770 - info: [ConfigProxy] Proxy API at http://localhost:8001/api/routes
21:47:17.939 - error: [ConfigProxy] Proxy error:  code=ECONNRESET
21:47:17.942 - error: [ConfigProxy] 503 GET /spawn/user/ApCsqQZgteFU/notebooks/widgets/examples/urth-pyspark-streaming.ipynb
00:40:17.180 - error: [ConfigProxy] Proxy error:  code=ECONNRESET
00:40:17.181 - error: [ConfigProxy] 503 GET /spawn/user/ApCsqQZgteFU/notebooks/widgets/examples/urth-pyspark-streaming.ipynb
00:40:17.183 - error: [ConfigProxy] Proxy error:  code=ECONNRESET
00:40:17.183 - error: [ConfigProxy] 503 GET /user/ApCsqQZgteFU/notebooks/widgets/examples/urth-pyspark-streaming.ipynb
00:40:17.405 - error: [ConfigProxy] Proxy error:  code=ECONNRESET
00:40:17.405 - error: [ConfigProxy] 503 GET /spawn/user/ApCsqQZgteFU/notebooks/index.ipynb

assert.js:89
  throw new assert.AssertionError({
  ^
AssertionError: false == true
    at ServerResponse.resOnFinish (_http_server.js:474:7)
    at emitNone (events.js:72:20)
    at ServerResponse.emit (events.js:166:7)
    at finish (_http_outgoing.js:529:10)
    at doNTCallback0 (node.js:407:9)
    at process._tickCallback (node.js:336:13)
```

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,4 +1,4 @@
-FROM node:latest
+FROM node:5-slim
 
 EXPOSE 8000
 

